go.mod

```diff
@@ -5,8 +5,12 @@ go 1.14
 require (
 	github.com/aws-controllers-k8s/runtime v0.6.0
 	github.com/aws/aws-sdk-go v1.38.11
+	github.com/ghodss/yaml v1.0.0
 	github.com/go-logr/logr v0.1.0
+	github.com/google/go-cmp v0.3.1
 	github.com/spf13/pflag v1.0.5
+	github.com/stretchr/testify v1.5.1
+	go.uber.org/zap v1.10.0 // indirect
```

Comment:
interesting, this an indirect dependency. `go mod tidy`?

Reply:
Typed “go mod tidy” into a shell, and the only change it created was removing the //indirect tag from go.uber.org/zap v1.10.0.
So now it just says go.uber.org/zap v1.10.0 without the indirect tag.